paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/action/ActionITCaseBase.java

```diff
@@ -136,14 +130,12 @@ protected void writeData(GenericRow... data) throws Exception {
         incrementalIdentifier++;
     }
 
-    protected List<String> getResult(TableRead read, List<Split> splits, RowType rowType)
-            throws Exception {
-        try (RecordReader<InternalRow> recordReader = read.createReader(splits)) {
-            List<String> result = new ArrayList<>();
-            recordReader.forEachRemaining(
-                    row -> result.add(DataFormatTestUtil.internalRowToString(row, rowType)));
-            return result;
+    protected void writeDataToDynamicBucket(List<GenericRow> data, int bucket) throws Exception {
```

Comment:
```suggestion
    protected void writeDataToBucket(List<GenericRow> data, int bucket) throws Exception {
```

This is only used in `CloneProcedureITCase`. Move it into that specific class.

Reply:
done